recipes/baseband/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "baseband" %}
+{% set version = "2.0.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5a5e386f1fbdd34c00802aeed10541d9facb31228df182f04f32244d240063b3
+  patches:
+    - baseband.patch
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+  noarch: python
+
+requirements:
+  host:
+    - astropy >=2.0
+    - pip
+    - python
+  run:
+    - astropy >=2.0
+    - python
+
+test:
+  imports:
+    - baseband
+    - baseband.dada
+    - baseband.data
+    - baseband.data.gsb
+    - baseband.gsb
+    - baseband.guppi
+    - baseband.helpers
+    - baseband.mark4
+    - baseband.mark5b
+    - baseband.vdif
+    - baseband.vlbi_base
+
+about:
+  home: https://baseband.readthedocs.io
+  license: GPLv3
+  license_family: GPL3
+  license_file: {{ RECIPE_DIR }}/LICENSE
```

Conda will automatically look in the recipe directory if it isn't found in the build directory:
```suggestion
  license_file: LICENSE
```